translator/src/main/scala/io/kaitai/struct/testtranslator/specgenerators/GoSG.scala

```diff
@@ -44,6 +45,16 @@ class GoSG(spec: TestSpec, provider: ClassTypeProvider) extends BaseGenerator(sp
   override def header() = {
     out.puts(s"func Test$className(t *testing.T) {")
     out.inc
+    out.puts("defer func() {")
+    out.inc
+    out.puts("if r := recover(); r != nil {")
+    out.inc
+    out.puts("debug.PrintStack()")
+    out.puts("t.Fatal(\"unexpected panic:\", r)")
+    out.dec
+    out.puts("}")
+    out.dec
+    out.puts("}()")
```

Comment:
This is actually a pretty significant change. Please add it to PR title.

Reply:
I honestly don't know what is it good for, I just concluded that given that most existing tests are using this and the `./spec_ruby_to_go` is also written in this manner, it's probably wise to do the same in the KST translator.

See commit https://github.com/kaitai-io/kaitai_struct_tests/commit/27e3785686db0f655287042727f14ff62954a67e. I must admit that I don't understand Go much.